sonic_platform_base/sonic_xcvr/api/public/c_cmis.py

```diff
@@ -814,7 +815,7 @@ def get_transceiver_status(self):
             trans_status['rxsigpowerhighwarning_flag'] = self.vdm_dict['Rx Signal Power [dBm]'][1][7]
             trans_status['rxsigpowerlowwarning_flag'] = self.vdm_dict['Rx Signal Power [dBm]'][1][8]
         except KeyError:
-            helper_logger.log_debug('fields not present in VDM')
+            logging.debug('fields not present in VDM')
```

Comment:
@xincunli-sonic is this log coming to syslog file by default? Could you check?

Reply:
Thanks @prgeor, confirmed this logging would be consider as discard, so, I pushed a new change with importing syslog. Also, this is a rare case, it should not impact other modules that are using syslog by race condition. Please help review it again.